Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -989,7 +1010,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     private func handleExpireRegistrationKey(completionHandler: ((Data?) -> Void)? = nil) {
         Task {
-            await rekey()
+            try? await rekey()
             completionHandler?(nil)
```

Comment:
I was a bit worried that we're suppressing an error value here, but I think this func is just used for the debug menu so should be fine.